### PR TITLE
Updated translations(issues raised by Q.A)

### PR DESCRIPTION
**Pull request by UbaidRajputt**

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ i18n
     resources: {
       en: {
         translations: {
+            "deviceVerificationSystem": "Device Verification System",
```
Comment (ikashifullah) [inline src/i18n.js]: It should be "Device Verification Subsystem"

File: src/i18n.js [hunk 2]
```diff
@@ -42,10 +43,19 @@ i18n
             "credentialMatch": "Credential Match",
             "wrongHttp" : "Wrong Http",
 
+            //Copyright
+            "copyright": "Copyright",
+            "allRightsReserved": "All Rights Reserved",
+            "Version": "Version",
+
+            //errors
+            "errors.FeildReq": " This field is required",
```
Comment (ikashifullah) [inline src/i18n.js]: Please correct spelling mistakes such as fieldReq, imei

**Review by ikashifullah — CHANGES_REQUESTED** [review]

File: src/i18n.js (hunk 1 above)
Comment (UbaidRajputt) [reply src/i18n.js]: Corrected

File: src/i18n.js (hunk 2 above)
Comment (UbaidRajputt) [reply src/i18n.js]: Corrected

**Review by ikashifullah — APPROVED** [review]

**Pull request merged**